barretenberg/cpp/src/barretenberg/vm2/tracegen/opcodes/get_env_var.test.cpp

```diff
@@ -0,0 +1,342 @@
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <cstdint>
+
+#include "barretenberg/vm2/common/aztec_constants.hpp"
+#include "barretenberg/vm2/common/aztec_types.hpp"
+#include "barretenberg/vm2/common/opcodes.hpp"
+#include "barretenberg/vm2/constraining/flavor_settings.hpp"
+#include "barretenberg/vm2/constraining/full_row.hpp"
+#include "barretenberg/vm2/testing/instruction_builder.hpp"
+#include "barretenberg/vm2/testing/macros.hpp"
+#include "barretenberg/vm2/tracegen/execution_trace.hpp"
+#include "barretenberg/vm2/tracegen/test_trace_container.hpp"
+
+namespace bb::avm2::tracegen {
+namespace {
+
+using enum ::bb::avm2::WireOpCode;
+
+class GetEnvVarTracegenTest : public ::testing::Test {
+  protected:
+    void SetUp() override
+    {
+        trace = std::make_unique<TestTraceContainer>();
+        builder = std::make_unique<ExecutionTraceBuilder>();
+        default_output = TaggedValue::from_tag(ValueTag::FF, FF(0));
+    }
+
+    std::unique_ptr<TestTraceContainer> trace;
+    std::unique_ptr<ExecutionTraceBuilder> builder;
```

Comment:
Neither of these need to be unique_ptrs, then your SetUp becomes almost trivial.